komodo-parent/pom.xml

```diff
@@ -536,49 +536,12 @@
 
 			<!--Inherited from BOM, but changes the default scope to "test" -->
 			<dependency>
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>${version.junit}</version>
-				<scope>test</scope>
-			</dependency>
-
-			<!--Inherited from BOM, but changes the default scope to "test" -->
-			<dependency>
 				<groupId>org.hamcrest</groupId>
 				<artifactId>hamcrest-library</artifactId>
 				<version>${version.org.hamcrest}</version>
 				<scope>test</scope>
 			</dependency>
 
-			<!--Inherited from BOM, but changes the default scope to "test" -->
-			<dependency>
-				<groupId>org.mockito</groupId>
-				<artifactId>mockito-core</artifactId>
-				<version>${version.org.mockito}</version>
-				<scope>test</scope>
-			</dependency>
-
-			<!--Inherited from BOM, but changes the default scope to "test" -->
-			<dependency>
-				<groupId>org.slf4j</groupId>
-				<artifactId>slf4j-api</artifactId>
-				<version>${version.org.slf4j}</version>
-			</dependency>
-
-			<!--Inherited from BOM, but changes the default scope to "test" -->
-			<dependency>
-				<groupId>org.slf4j</groupId>
-				<artifactId>slf4j-log4j12</artifactId>
-				<version>${version.org.slf4j}</version>
-			</dependency>
-
-			<!--Inherited from BOM, but changes the default scope to "test" -->
-			<dependency>
-				<groupId>log4j</groupId>
-				<artifactId>log4j</artifactId>
-				<version>${version.log4j}</version>
-			</dependency>
-
```

Comment:
I assume there is a reason these have been removed. Can you specify please?

Reply:
I would assume that you do not want to generally make log4j a compile dependency correct?

Reply:
Understand. Again, a quick line in the commit message, please?